src/components/Timeline/Grid/index.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import getGridRowCells from '../../../utils/getGridRowCells'
+
+const Grid = ({ time, timebar }) =>
+  <div className="rt-grid">
+    {getGridRowCells(timebar).map(({ id, start, end }) =>
```

Comment:
This component shouldn't be provided with the whole `timebar` (it shouldn't even know what a `timebar` is!). Instead `getGridCells` should be called a few layers higher. Sometimes a bit of a smell when you pass a load of data to a function for it to then have to filter out and find the bits it needs (when the filtering could have been done at a higher level). I.e. would expect 

```jsx
<Body time={time} grid={grid} tracks={tracks} clickElement={clickElement} />
```

In `src/components/Timeline/index.jsx`